denizen_scripts/hub1/repo-link/Adriftus_Moderator_Panel/panels/mod_ban_inv.dsc

```diff
@@ -0,0 +1,51 @@
+# -- BAN PANEL
+mod_ban_inv:
+  type: inventory
+  debug: false
+  title: <&6>A<&e>MP <&f>› <&c>Ban <&6><player.flag[amp_map].as_map.get[uuid].as_player.name> from <player.flag[amp_map].as_map.get[from].to_titlecase>.
+  inventory: CHEST
+  size: 54
+  definitions:
+    x: <item[air]>
+    b1: <item[yellow_stained_glass_pane].with[display_name=<&r>]>
+    b2: <item[orange_stained_glass_pane].with[display_name=<&r>]>
+    back: <item[red_stained_glass_pane].with[display_name=<&c><&l>↩<&sp>Actions<&sp>panel;nbt=<list[to/actions]>]>
+    head: <item[mod_player_item]>
+  procedural items:
+    - define inventory:<list>
+    - foreach <list[1|2|3]> as:level:
+      - foreach <script[mod_ban_infractions].list_keys[<[level]>]> as:infraction:
+        - define item <item[mod_level<[level]>_item]>
+        - define name <[item].script.parsed_key[tag]><&sp><[infraction]>
+        - define lore <list[<&b>Level<&co><&sp><[item].script.parsed_key[colour]><[level]>]>
+        - define lore:|:<&c>Right<&sp>Click<&sp>to<&sp>ban<&co>
+        - define lore:|:<player.flag[amp_map].as_map.get[uuid].as_player.name>
+        - define lore:|:<&c>Clic<&sp>Droit<&sp>pour<&sp>bannir<&co>
+        - define nbt <list[LEVEL/<[level]>|INFRACTION/<[infraction]>|LENGTH/<script[mod_ban_infractions].data_key[<[level]>.<[infraction]>.length]>]>
+        - define item <[item].with[display_name=<[name]>;lore=<[lore]>;nbt=<[nbt]>]>
+        - define inventory:|:<[item]>
```

Comment:
This value being inserted into the list is a single list value, `:->:` is preferred for single entry insertions over `:|:`